### PR TITLE
chore: update issue templates

### PR DESCRIPTION
**Pull request by YoshinoriN**
## What does it do?

Create new GitHub issue template.
New issue template may help users & contributors. Also maintainers.

> [Creating issue templates for your repository](https://help.github.com/en/articles/creating-issue-templates-for-your-repository)

## How to test

Nothing

## Screenshots

Nothing

## Pull request tasks

- [ ] Add test cases for the changes.
- [ ] Passed the CI test.

## Others

I do not set a default label. Because issue author cannot define certainly which issue types. So, we should triage an issue later.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+labels: ''
+assignees: ''
+
+---
+
+<!-- NOTE:
+
+If you find that markdown files are not rendered as expected, please go to https://marked.js.org/demo/ to see if it can be reproduced there. If it can be reproduced, please file a bug to https://github.com/markedjs/marked.
+
+If you want help on your bug, please also send us the github repository where your hexo code is stored. If it is not on github, please post it on github, it would greatly help.
```
Comment (weyusi) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Must be github? I personally don't mind if reporter is using others (gitlab, bitbucket, etc), as long it's git and public.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+-->
+
+## Check List
+
+Please check followings before submitting a new issue.
+
+- [ ] I already read [Docs page](https://hexo.io/docs/) & [Troubleshooting page](https://hexo.io/docs/troubleshooting)
```
Comment (weyusi) [inline .github/ISSUE_TEMPLATE/bug_report.md]: `I have` is more common, applies to the rest.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (YoshinoriN) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I just copied this sentence from the current issue template. I will fix. Thanks :)

**Comment by YoshinoriN** [comment]
Thanks, @weyusi  :)
I fixed your review point. But, please do not merge yet. I wanna ask to other maintainer agree or not agree.

@hexojs/core 
How do you think? I believe the new issue template help us, but users may shrink.

**Comment by tcrowe** [comment]
I think it will help users and hexo team. Thank you @YoshinoriN and @weyusi 👍

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+labels: ''
+assignees: ''
+
+---
+
+<!-- NOTE:
+
+If you find that markdown files are not rendered as expected, please go to https://marked.js.org/demo/ to see if it can be reproduced there. If it can be reproduced, please file a bug to https://github.com/markedjs/marked.
+
+If you want help on your bug, please also send us the git repository (GitHub, GitLab, Bitbucket etc...) where your hexo code is stored. It would greatly help.
```
Comment (weyusi) [inline .github/ISSUE_TEMPLATE/bug_report.md]: suggest to add, "If you prefer not to have your hexo code out in public, please upload to a private GitHub repository and grant read-only access to hexojs/core".

Comment (YoshinoriN) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I will update later :)

Comment (YoshinoriN) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done :)

**Comment by YoshinoriN** [comment]
I delete `[WIP]` comment. Please approve & merge this PR if there are no other problems.

Thanks :)

**Comment by YoshinoriN** [comment]
@curbengh @tcrowe @tomap Thanks!

**Pull request merged**